api/management/commands/prepare-admin-geojson.py

```diff
@@ -0,0 +1,27 @@
+import subprocess
+import os
+from django.core.management.base import BaseCommand, CommandError
+
+class Command(BaseCommand):
+  help = "This command produces a countries.geojson and districts.geojson to be uploaded to Mapbox as a tileset. It is the source for all GO Maps."
+
+  def handle(self, *args, **options):
+    try:
+      DB_HOST = os.environ['DJANGO_DB_HOST']
```

Comment:
you can probably get these a bit nicer by:

```
    from django.conf import settings
    db = settings.DATABASES[0]
    DB_USER = db.USER
    etc...
```

Probably a bit better to take these from settings than from the environ.

Reply:
Oh that's a great idea. Yes.